src/styles/blocks/product-card.scss

```diff
@@ -0,0 +1,89 @@
+.product-card {
+  display: flex;
+  flex-direction: column;
+  width: 200px;
+  padding: 32px 16px 16px;
+  border: 1px solid $border-card-color;
+  border-radius: 5px;
+  background-color: $main-color;
+  color: $secondary-color;
+
+  &__image-wrapper {
+    width: 160px;
+    height: 134px;
+    margin: 0 3px 40px;
+  }
+  
+  &__image {
+    width: 100%;
+    height: 100%;
+  }
+  
+  &__product-title {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: $main-accent-color;
+    margin-bottom: 4px;
+  }
+
+  &__product-code {
+    margin-bottom: 16px;
+  }
+  
+  &__review-wrapper {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 24px;
+  }
+  
+  &__review-number {
+    color: $main-accent-color;
+  }
+  
+  &__price {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
+    margin-bottom: 16px;
+  }
+  
+  &__price-lable {
+    font-size: 12px;
+    line-height: 18px;
+    color: $secondary-color;
+  }
+  
+  &__price-number {
+    color: $main-accent-color;
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+  }
+  
+  &__button-link {
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    border-radius: 5px;
+    background-color: $blue-accent-color;
+    //padding: 11px 68px;
+    height: 40px;
```

Comment:
it's not a good practice to hardcode height for elements with content inside. just use padding